src/ComposerEventHandler.php

```diff
@@ -194,14 +194,19 @@ function ($file) use ($options) {
 
     private function updateFile(string $source, string $destination, bool $silentOverride = false): void
     {
+        if (!file_exists($destination)) {
```

Comment:
That's not very convenient. In order to override a config you need to manually create a `config/packages/vendor/package-name/config-name.php` and then do `composer update`.

That won't create dist file either so there's no way to know which file to create and where to put it.